aws/resource_aws_cognito_user_pool_domain.go

```diff
@@ -119,6 +136,7 @@ func resourceAwsCognitoUserPoolDomainRead(d *schema.ResourceData, meta interface
 	desc := domain.DomainDescription
 
 	d.Set("domain", d.Id())
+	d.Set("certificate_arn", desc.CustomDomainConfig.CertificateArn)
```

Comment:
To prevent potential panics, we should ensure `CustomDomainConfig` is not `nil` first, e.g.

```suggestion
	d.Set("certificate_arn", "")
	if desc.CustomDomainConfig != nil {
		d.Set("certificate_arn", desc.CustomDomainConfig.CertificateArn)
	}
```